package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "auth-final",
+  "version": "0.1.0",
+  "private": true,
+  "proxy": "http://localhost:5000",
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.8.5",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
+    "cors": "^2.8.5",
+    "evergreen-ui": "^6.10.4",
```

Comment:
What is this evergreen-ui used for?

Reply:
I have used it for the File picker in the form.